shared/js/media/jpeg_metadata_parser.js

```diff
@@ -94,6 +95,7 @@ function parseJPEGMetadata(file, metadataCallback, metadataError) {
 
       case 0xE1:  // APP1 segment. Probably holds EXIF metadata
         parseAPP1(data);
+        break;
```

Comment:
You can't add a break here. The comment that follows this line indicates that this is an intentional fallthough to the default case. I suppose you might have to add a comment that turns of jshint warnings on this line.  I'm not sure what that comment is.